### PR TITLE
fixing the function to deal with JSON format

### PR DESCRIPTION
**Pull request by san89**
1. Deal with JSON format on different machines
2. Fix merge function. It does not recognise sort=True argument

### CODE REVIEW

File: google_analytics/preprocessing.py [hunk 1]
```diff
@@ -20,7 +20,10 @@ def load(path, nrows=None):
 
     # Normalize JSON columns
     for column in JSON_COLUMNS:
-        column_as_df = pd.io.json.json_normalize(df[column])
+        if format_solver:
```
Comment (Dimkoim) [inline google_analytics/preprocessing.py]: What excatly the `format_solver` does and when should I use it?

Comment (san89) [reply google_analytics/preprocessing.py]: In some laptops, the command  `pd.io.json.json_normalize` does not work to change the format to JSON. A way to solve it is to apply the format to every row as is proposed on the format solver.

File: google_analytics/preprocessing.py [hunk 2]
```diff
@@ -44,7 +47,7 @@ def process(train, test):
     del train["transactionRevenue"]
 
     train_len = train.shape[0]
-    merged = pd.concat([train, test], sort=False)
+    merged = pd.concat([train, test])
```
Comment (joepvdbogaert) [inline google_analytics/preprocessing.py]: We need sort=false; otherwise the train and test sets get mixed up.

**Comment by san89** [comment]
The issue was solved updating Pandas

**Pull request closed without merging**